Web/starter_project_barbies/store/about/about-api.ts

```diff
@@ -0,0 +1,28 @@
+import { ProblemItem, Project, Session, SolutionItem } from '@/types/about'
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+
+export const aboutApi = createApi({
+  reducerPath: 'about',
+  baseQuery: fetchBaseQuery({ baseUrl: 'http://localhost:5000/' }),
```

Comment:
Move the base url to a .env file to keep things consistent with the rest of the team.